domains/authorization/Authorization.API/API/Models/Organization.cs

```diff
@@ -31,6 +34,9 @@ private Organization()
     public bool TermsAccepted { get; private set; }
     public int? TermsVersion { get; private set; }
     public DateTimeOffset? TermsAcceptanceDate { get; private set; }
+    public bool ServiceProviderTermsAccepted { get; private set; }
+    public int? ServiceProviderTermsVersion { get; private set; }
+    public DateTimeOffset? ServiceProviderTermsAcceptanceDate { get; private set; }
```

Comment:
Should we align on using the UnixTimestamp class that is used many other places and slowly move away from DateTimeOffset? @martinhenningjensen @tnickelsen

Reply:
Det er ikke nogen større opgave :) Jeg har lige lavet det. Hvis du stadig ikke syntes det skal med kan du reverte mit sidste commit. https://github.com/Energinet-DataHub/energy-origin/pull/1605/commits/c33cee49702d70a9475fd381d772f3532794d231

Vi har allerede converters i shared, så det er bare at bruge dem :D Har også kørt test igennem for at sikre mig der ikke er ødelagt noget.

Reply:
AMAZING

Reply:
Så vi ændrer ikke feltet i databasen til longs?

Reply:
Jeg har set ikke set jeres tråd her, det lyder som en god ide at aligne omkring UnixTimeStamp som value object til tids stempler. Jeg har lavet converters så man kan benytte UnixTimeStamp uanset om man bruger long eller Timestamp i databasen.

Reply:
Can this be resolved?